pygen/pygen_src/test/riscv_instr_cov_test.py

```diff
@@ -116,6 +116,8 @@ def get_coverage_report():
         file.write(tabulate(table, headers, tablefmt="grid",
                             numalign="center", stralign="center"))
         file.close()
+        # Write in xml format to be read by pyucis-viewer (visulization)
+        vsc.write_coverage_db("cov_db.xml")
```

Comment:
just to confirm, after we save this `cov_db.xml`, is the user able to open this with UCIS-viewer from the command line?

Reply:
That is correct, yes. Basically what happens is that we add an additional command line option to the `cov.py` file for visualization purpose: if the flag is enabled, then this `cov_db.xml` will be generated, then as you said, the user will be able to read this xml file with UCIS-Viewer from the command line and see the output (alternatively, we can embed this functionality as well but in that case we need to add the `pyucis-viewer` package to the `requirements.txt` file, and also include the source folder of `pyucis-viewer` in the `riscv-dv/pygen` directory, which might not be a good idea at the moment. With that said, I suggest to leave it for user to read the file using pyucis-viewer). What do you think?

Reply:
got it. I think this is fine as is for right now - if we ever get to the point where we are comfortable integrating `pyucis-viewer` into the end-to-end run flow, we can add functionality at that later time.

Reply:
Makes sense, sure :-)